lib/inspec/base_cli.rb

```diff
@@ -69,6 +69,12 @@ def self.target_options # rubocop:disable MethodLength
         desc: 'Use SSL for transport layer encryption (WinRM).'
       option :self_signed, type: :boolean,
         desc: 'Allow remote scans with self-signed certificates (WinRM).'
+      option :transport, type: :string,
+        desc: 'Specify which transport to use, defaults to negotiate (WinRM).'
```

Comment:
I suggest we add a list of valid transport names.  Also, the help text says it defaults to `:negotiate`, but the Thor option doesn't have a default (perhaps omission causes the WinRM library to negotiate - but if so, a comment would be good)

Reply:
resolved.